<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterVente</class>
 <widget class="QDialog" name="ajouterVente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_affiche_vente">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:	15px;border:2px solid black;
background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Consulter les ventes</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>20</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/WhatsApp Image 2024-04-30 à 20.55.44_6b270705.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonProfil">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;border:2px solid black;
background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/devenir-pharmacien-921x1024.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>320</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);font-size:20px;
font: 20pt &quot;Segoe Script&quot;;color:rgb(170, 85, 255);border:2px solid black;</string>
   </property>
   <property name="text">
    <string>PHARMACIE N.M.B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);font-size:15px;
font: 600 16pt &quot;Segoe UI Variable Text Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Dr.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>10</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid black;color:white;border-radius:10px;
font: 900 12pt &quot;Arial Black&quot;;
background-color: rgb(255, 114, 116);</string>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ajouter_vente">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(204, 204, 204);;font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Ajouter Vente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Cherecher Médicament</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/OIP (3).jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Consulter_stock">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Consulter Stock</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/R (1).png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>0</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="nome_user">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
font: 900 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>16</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid blue;border-radius:4px;
font: 10pt &quot;Rockwell&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>911</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);border:2px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>511</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 244, 216);</string>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_code_med">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 14pt &quot;Arial Black&quot;;color: rgb(16, 88, 255);</string>
    </property>
    <property name="text">
     <string>Libellé médicament</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Libell">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 14pt &quot;Arial Black&quot;;
color: rgb(16, 88, 255);

</string>
    </property>
    <property name="text">
     <string>Code médicament</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 14pt &quot;Arial Black&quot;;color: rgb(16, 88, 255);</string>
    </property>
    <property name="text">
     <string>Quantité</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_quantite">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Id_Client">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>59</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>72</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 14pt &quot;Arial Black&quot;;
color: rgb(16, 88, 255);
</string>
    </property>
    <property name="text">
     <string>ID Client</string>
    </property>
   </widget>
   <zorder>label_10</zorder>
   <zorder>lineEdit_code_med</zorder>
   <zorder>lineEdit_Libell</zorder>
   <zorder>label_9</zorder>
   <zorder>label_11</zorder>
   <zorder>spinBox_quantite</zorder>
   <zorder>lineEdit_Id_Client</zorder>
   <zorder>label_12</zorder>
  </widget>
  <widget class="QPushButton" name="pushButton_Abandonner">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>510</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
color:rgb(255, 255, 255);
background-color: rgb(255, 135, 99);</string>
   </property>
   <property name="text">
    <string>Abandonner</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Valider_Vente">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(94, 255, 193);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_FiVente_Affiche_facture">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>510</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(215, 69, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>         FIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>520</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(191, 112, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/invoice-vector-icon.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_affiche_vente</zorder>
  <zorder>line_4</zorder>
  <zorder>label_4</zorder>
  <zorder>ButtonProfil</zorder>
  <zorder>label_8</zorder>
  <zorder>label</zorder>
  <zorder>label_6</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_ajouter_vente</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton_Consulter_stock</zorder>
  <zorder>label_7</zorder>
  <zorder>line</zorder>
  <zorder>nome_user</zorder>
  <zorder>line_2</zorder>
  <zorder>label_date</zorder>
  <zorder>pushButton_Abandonner</zorder>
  <zorder>pushButton_Valider_Vente</zorder>
  <zorder>pushButton_FiVente_Affiche_facture</zorder>
  <zorder>label_13</zorder>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
